examples/UKCore-Specimen-BloodSpecimen-Example.xml

```diff
@@ -1,29 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Specimen xmlns="http://hl7.org/fhir">
- <id value="UKCore-Specimen-BloodSpecimen-Example"/>
- <status value="available"/>
- <type>
-  <coding>
-   <system value="http://snomed.info/sct"/>
-   <code value="119297000"/>
-   <display value="Blood specimen"/>
-  </coding>
- </type>
- <subject>
-  <reference value="Patient/UKCore-Patient-RichardSmith-Example"/>
- </subject>
- <receivedTime value="2016-02-05T11:10:00Z"/>
- <collection>
-  <collector>
-   <reference value="Practitioner/UKCore-Practitioner-ConsultantSandraGose-Example"/>
-  </collector>
-  <collectedDateTime value="2016-02-09T16:00:00Z"/>
-  <method>
-   <coding>
-    <system value="http://snomed.info/sct"/>
-    <code value="278450005"/>
-    <display value="Finger-prick sampling (procedure)"/>
-   </coding>
-  </method>
- </collection>
+	<id value="UKCore-Specimen-BloodSpecimen-Example"/>
+	<status value="available"/>
+	<type>
+		<coding>
+			<system value="http://snomed.info/sct"/>
+			<code value="119297000"/>
+			<display value="Blood specimen"/>
+		</coding>
+	</type>
+	<subject>
+		<reference value="Patient/UKCore-Patient-RichardSmith-Example"/>
+	</subject>
+	<receivedTime value="2016-02-05T11:10:00Z"/>
+	<collection>
+		<collector>
+			<reference value="Practitioner/UKCore-Practitioner-ConsultantSandraGose-Example"/>
+		</collector>
+		<collectedDateTime value="2016-02-09T16:00:00Z"/>
+		<method>
+			<coding>
+				<system value="http://snomed.info/sct"/>
+				<code value="28520004"/>
+				<display value="Venipuncture for blood test"/>
+			</coding>
+		</method>
+		<bodySite>
+			<!-- Use of the bodySite R5 backport extension to reference a BodyStucture, in this case, the draw site. -->
+			<extension url="https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-BodySiteReference">
+				<valueReference>
+					<reference value="BodyStructure/UKCore-BodyStructure-MedianCubitalVeinExample"/>
+				</valueReference>
+			</extension>
+			<!-- Use of the bodySite element to reference a codeable concept, in this case, the draw site. -->
+			<coding>
```

Comment:
Given we're trying to replicate CodeableReference from R5, doesn't that only allow 1 or the other? I'd be tempted to remove the codeable concept here